analyses/focal-cn-file-preparation/06-find-recurrent-calls.Rmd

```diff
@@ -83,17 +83,27 @@ The `recurrence_cutoff` defined as 3 appears to be reasonable as seen in the dis
 
 ```{r}
 # Create a data frame of only observations that have counts greater than the `recurrence_cutoff`
-final_df_recurrent <- final_df_tally %>%
+recurrent_df <- final_df_tally %>%
   filter(count > recurrence_cutoff)
 
 # Display table
-final_df_recurrent
+recurrent_df
+```
+
+### Filter focal CN data frame to include only recurrent observations
+
+```{r}
+# Filter the focal CN data frame to include only the recurrent observations
```

Comment:
I don't think this is exactly what you want. I * think* you want to have `Kids_First_Biospecimen_ID` to be in your `group_by()` step in the first place. That way the tally's are also by sample.

Reply:
If I do that, the count will be 1 for all observations. I _believe_ we want to find out which status calls occur in multiple samples for a specific region.

Reply:
Ah I see. Yes you're right. I lost track of the goal here.